package.json

```
@@ -9,7 +9,7 @@
     "watch-ts": "tsc -w",
     "build": "node esbuild/esbuild.config.js --build",
     "start": "node $NODE_OPTIONS dist/server.js | bunyan -o short",
-    "start:dev": "concurrently -k -p \"[{name}]\" -n \"ESBuild,Node\" -c \"yellow.bold,cyan.bold\" \"node esbuild/esbuild.config.js --build --watch\" \"node esbuild/esbuild.config.js --dev-server --inspect \"",
+    "start:dev": "concurrently -k -p \"[{name}]\" -n \"ESBuild,Node,WireMock\" -c \"yellow.bold,cyan.bold\" \"node esbuild/esbuild.config.js --build --watch\" \"node esbuild/esbuild.config.js --dev-server --inspect \" \"npm run wiremock\"",
```

Very very minor thing - you're specifying 3 concurrent processes here, but only 2 log colours.  To be honest it probably simplifies things to just remove the colour bit:
```suggestion
    "start:dev": "concurrently -k -p \"[{name}]\" -n \"ESBuild,Node,WireMock\" \"node esbuild/esbuild.config.js --build --watch\" \"node esbuild/esbuild.config.js --dev-server --inspect \" \"npm run wiremock\"",
```



If you want to keep the colours, just set a colour for the WireMock logs too e.g.
```suggestion
    "start:dev": "concurrently -k -p \"[{name}]\" -n \"ESBuild,Node,WireMock\" -c \"yellow.bold,cyan.bold,magenta.bold\" \"node esbuild/esbuild.config.js --build --watch\" \"node esbuild/esbuild.config.js --dev-server --inspect \" \"npm run wiremock\"",
```